pulsar-io/core/src/main/java/org/apache/pulsar/io/core/ConnectorContext.java

```
@@ -68,7 +69,15 @@
      * @return the namespace this source belongs to
      */
     String getNamespace();
-    
+
+    /**
+     * Get subscription type
+     * @return subscription type
+     */
+    default SubscriptionType getSubscriptionType() {
+        return null;
```

Can we throw "UnsupportedOperationException" ?